installer/data/mysql/kohastructure.sql

```diff
@@ -823,6 +823,25 @@ CREATE TABLE `import_items` (
   KEY `branchcode` (`branchcode`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
 
+--
+-- Table structure for table 'invoice_adjustments'
+--
+
+DROP TABLE IF EXISTS invoice_adjustments;
+CREATE TABLE IF NOT EXISTS invoice_adjustments (
```

Comment:
As above, prepend with `aq_` to keep it consistent with the rest of acquisitions